tools/pyboard.py

```diff
@@ -482,13 +482,11 @@ def fname_remote(src):
         return src
 
     def fname_cp_dest(src, dest):
-        src = src.rsplit("/", 1)[-1]
+        src = os.path.basename(src)
```

Comment:
I guess this fixes a bug where it wouldn't work on Windows using `\` as separators?